examples/scripts/flux_batch_job.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Read all the hostname the job is running
```

Comment:
Nice job here! :raised_hands: I love how you used flux exec, and your bash skillz are Napolean dynamite level awesome.